src/Sarif/VersionConstants.cs

```diff
@@ -1,12 +1,13 @@
-// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT        
-// license. See LICENSE file in the project root for full license information. 
```

Comment:
This file _was_ mistakenly checked in, but now we're _generating_ it here, so it's all good.